app/middlewares/trackUser.ts

```diff
@@ -1,9 +1,12 @@
 import * as ReactGA from "react-ga";
+import * as store from "store";
 import * as Sentry from "@sentry/browser";
 import { ACTION_TYPES } from "../actions/actionTypes";
 import { CurrentUser } from "../model/currentUser";
 import EnvChecker from "../helpers/envChecker";
 
+export const USER_ID_KEY = "u_id";
```

Comment:
Is there any reason to define `USER_ID_KEY` in this module instead of actionTicketManager helper (like other keys, `DEVICE_ID_KEY` and `SESSION_ID_KEY`)?

Reply:
To avoid actionTicketManager file is compiled to the server-side rendering bundle.
It can break the entire bundle because of the `window` object.

Reply:
However, I make the `window` object don't be exposed anymore.